### PR TITLE
Translate Fragements page

### PR DESCRIPTION
**Pull request by ph1p**

### CODE REVIEW

File: content/docs/fragments.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Fragments
 permalink: docs/fragments.html
 ---
 
-A common pattern in React is for a component to return multiple elements. Fragments let you group a list of children without adding extra nodes to the DOM.
+Es kommt häufig vor, dass eine Komponente mehrere Elemente. Mit Fragments kannst du eine Liste von Kindelementen gruppieren, ohne dem DOM einen zusätzlichen Knoten hinzuzufügen.
```
Comment (tamarau) [inline content/docs/fragments.md]:
```suggestion
Es kommt häufig vor, dass eine Komponente mehrere Elemente zurückgibt. Mit Fragments kannst du eine Liste von Kindelementen gruppieren, ohne dem DOM einen zusätzlichen Knoten hinzuzufügen.
```

File: content/docs/fragments.md [hunk 2]
```diff
@@ -38,93 +38,93 @@ class Table extends React.Component {
...
+    <td>Welt</td>
   </tr>
 </table>
 ```
 
-### Short Syntax {#short-syntax}
+### Kurzschreibweise {#short-syntax}
 
-There is a new, shorter syntax you can use for declaring fragments. It looks like empty tags:
+Es gibt eine neue kürzere Schreibweise, welche du verwenden kannst um Fragmente zu deklarieren. Diese sieht aus wie leere Tags:
```
Comment (tamarau) [inline content/docs/fragments.md]:
```suggestion
Es gibt eine neue kürzere Schreibweise, welche du verwenden kannst um Fragmente zu deklarieren. Diese sieht wie leere Tags aus:
```
...so it sounds less 'Denglisch' syntax-wise :)

**Comment by EyMaddis** [comment]
It seems I did not hit send on my PR review...

**Comment by ph1p** [comment]
Thanks @EyMaddis ! (:

**Comment by EyMaddis** [comment]
My PR does not mean that @tamarau's is not as valid. (Scroll up a bit, @ph1p ) :)

**Comment by ph1p** [comment]
> Hi,
> Tried myself with reviewing your PR for the first time, so please feel free to use my suggestions or not and maybe get a second person to look over the PR ☺️

I really appreciate it! I try to translate as much as possible at the react docs. Nothing is carved in stone and everytime something is wrong, a person can open a new PR and correct it.

**Comment by ph1p** [comment]
> My PR does not mean that @tamarau's is not as valid. (Scroll up a bit, @ph1p ) :)

I've seen it ^^ Thanks

**Pull request merged**